policykit/policyengine/models.py

```diff
@@ -1176,81 +1200,46 @@ class BasePolicy(models.Model):
     )
     """The community which the policy belongs to."""
 
-    # FIXME: Should not allow this to be null.
-    name = models.TextField(null=True, blank=True)
+    name = models.CharField(max_length=100)
     """The name of the policy."""
 
     description = models.TextField(null=True, blank=True)
     """The description of the policy. May be empty."""
 
-    is_bundled = models.BooleanField(default=False)
-    """True if the policy is part of a bundle. Default is False."""
-
     is_active = models.BooleanField(default=True)
     """True if the policy is active. Default is True."""
 
     modified_at = models.DateTimeField(auto_now=True)
     """Datetime object representing the last time the policy was modified."""
 
+    # TODO(https://github.com/amyxzhang/policykit/issues/341) add back support for policy bundles
+    bundled_policies = models.ManyToManyField("self", blank=True, symmetrical=False, related_name="member_of_bundle")
+    """Policies bundled inside this policy."""
+
```

Comment:
Policies can have other policies within them, and they can also be part of a policy bundle. That's why this is an asymetrical ManyToMany field. It can be used like this:

```
p = Policy.objects.all()[0]
p1 = Policy.objects.all()[1]

p.bundled_policies.add(p1)
p.bundled_policies.count() # 1, it has one bundled policy
p.member_of_bundle.count() # 0, it is not a member of any bundles

p1.bundled_policies.count() # 0, it doesn't have any bundled policies
p1.member_of_bundle.count() # 1, it is a member of one bundle

```